editoast/src/views/infra/mod.rs

```diff
@@ -561,6 +565,62 @@ pub mod tests {
         assert!(Infra::delete(db_pool, cloned_infra_id).await.unwrap());
     }
 
+    #[rstest]
+    async fn infra_with_errors_clone(
+        db_pool: Data<DbPool>,
+        #[future] empty_infra: TestFixture<Infra>,
+    ) {
```

Comment:
Why not use a filled infra like `small_infra`. You will have to generate layers but you can check that all objects are well cloned.
This will test more things with fewer manual things to do.